library/alloc/tests/vec.rs

```diff
@@ -880,7 +880,7 @@ fn test_from_iter_partially_drained_in_place_specialization() {
 #[test]
 fn test_from_iter_specialization_with_iterator_adapters() {
     fn assert_in_place_trait<T: InPlaceIterable>(_: &T) {};
-    let src: Vec<usize> = vec![0usize; 65535];
+    let src: Vec<usize> = vec![0usize; if cfg!(miri) { 256 } else { 65535 }];
```

Comment:
The size does not really matter to that test, it was chosen arbitrarily. You can lower it unconditionally.

Reply:
Thanks! Done that.